ogcapi-java-dev-td.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Is there way to integrate docker-compose.yml together, cost dev need to maintain two very similar config one for docker one for aws tasks